blosc-sys/Cargo.toml

```diff
@@ -11,3 +11,4 @@ Rust FFI bindings to the C-BLOSC compression library
 documentation = "https://docs.rs/blosc-sys"
 categories = ["external-ffi-bindings"]
 keywords = ["compression"]
+links = "blosc"
```

Comment:
What does this do?

Reply:
With this attribute cargo only allows a single build of `blosc` native library (through `blosc-sys`) is included in the same build. If the user has requested two incompatible version it will fail at package resolution time instead of at link time. C has global symbols which aren't versioned like rust